XenAdmin/Wizards/PatchingWizard/PlanActions/ApplyXenServerPatchPlanAction.cs

```diff
@@ -76,12 +76,20 @@ protected override void RunWithSession(ref Session session)
                 AddProgressStep(string.Format(Messages.UPDATES_WIZARD_APPLYING_UPDATE, xenServerPatch.Name,
                     host.Name()));
 
+                PatchPrecheckOnHostPlanAction.RefreshUpdate(host, mapping, session);
+
                 XenRef<Task> task = null;
 
                 if (mapping is PoolPatchMapping patchMapping)
+                {
+                    log.InfoFormat("Allpying patch on '{0}'. Patch{1}' (uuid = '{2}'; opaque_ref = '{3}')", host.Name(), patchMapping.Pool_patch.Name(), patchMapping.Pool_patch.uuid, patchMapping.Pool_patch.opaque_ref);

```

Comment:
"Allpying" -> "Applying"

Reply:
Thanks @kc284 , I've updated the last commit